src/main/java/de/codingair/clansystem/utils/clan/Clan.java

```diff
@@ -128,15 +326,14 @@ public UUID getMemberByRank(Rank rank) {
         } else return null;
     }
 
-    public boolean hasPermission(UUID uuid, Permission permission) {
-        Rank r = getRank(uuid);
-        return r != null && r.hasPermission(permission);
-    }
-
     public Rank getRank(UUID uuid) {
```

Comment:
I would like to see this to have no conflict
```suggestion
    public Rank getMemberRank(UUID uuid) {
```